spec/rails_helper.rb

```diff
@@ -61,3 +61,18 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :minitest
+    with.library :rails
+  end
+end
+
+
+VCR.configure do |config|
```

Comment:
It's good to see you started using VCR, but personally I would rather stick with Webmock & manually create fixture data first before using VCR, if you're struggling with setup stuff. Largely because VCR is quite configuraiton-heavy, and if I can get it working with Webmock, then that's really all I need to test stuff! 
Think of VCR as more of a stretch goal in this case.